connect/api/v1/recent_activity/views.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import views, status
+from rest_framework.response import Response
+
+from django.contrib.auth import get_user_model
+
+from connect.api.v1.internal.permissions import ModuleHasPermission
+from connect.common.models import Project, RecentActivity
+
+User = get_user_model()
+
+
+class RecentActivityAPIView(views.APIView):
+    permission_classes = [ModuleHasPermission]
+
+    def post(self, request):
+        if request.data.get("flow_organization"):
+            project = Project.objects.filter(flow_organization=request.data.get("flow_organization"))
+        else:
+            project = Project.objects.filter(uuid=request.data.get("project_uuid"))
+        if len(project) > 0:
```

Comment:
After if, for and among other logic blocks use a blank line
More details: https://peps.python.org/pep-0008/#blank-lines